tests/manifest.ttl

```diff
@@ -1,17 +1,27 @@
-## Distributed under both the "W3C Test Suite License" [1]
-## and the "W3C 3-clause BSD License".
 ## [1] https://www.w3.org/Consortium/Legal/2008/04-testsuite-license
 ## [2] https://www.w3.org/Consortium/Legal/2008/03-bsd-license
 
-PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
-PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#>
-PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#>
-PREFIX rdft:   <http://www.w3.org/ns/rdftest#>
-PREFIX trs:    <https://w3c.github.io/rdf-star/tests#>
+@PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
+@PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@PREFIX rdft:   <http://www.w3.org/ns/rdftest#> .
+@PREFIX trs:    <https://w3c.github.io/rdf-star/tests#> .
+@PREFIX dct: <http://purl.org/dc/terms/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
 
 trs:manifest  rdf:type mf:Manifest ;
-    rdfs:label "RDF-star test suite" ;
-    rdfs:seeAlso <https://w3c.github.io/rdf-tests/> ;
+  rdfs:label "RDF-star test suite"@en ;
+	rdfs:label "La suite des tests pour RDF-star"@fr;
+	rdfs:label "Conjuto de pruebas para RDF-star"@es;
+	dct:issued "2021-06-21"^^xsd:date ; #not sure if this was the first release date
+	dct:creator [ foaf:name "Gregg Kellog" ] ; #maybe add foaf:homepage
+	dct:creator [ foaf:name "Andy Seaborne" ];
+  rdfs:seeAlso <https://w3c.github.io/rdf-tests/> ;
+	dct:modified "2021-07-18"^^xsd:date ; # change to the exact date
+	dct:licence <https://www.w3.org/Consortium/Legal/2008/04-testsuite-license>;
+	dct:licence <https://www.w3.org/Consortium/Legal/2008/03-bsd-license> ;
```

Comment:
:-)

RDF is conjunctive - these two `dct:licence` that say _both_ licenses apply, not a choice of one of them!

The root LICENSE is "Contributions to Test Suites are made under the W3C 3-clause BSD License" so drop the testsuite-license as it does not apply.

Background:
The testsuite-license is not an open source license (there is no unrestricted right to modify granted). The idea being test suite for working group is what is produced, not a downstream modified form there of. Hence the bsd-license, which is open source.

Reply:
ACK! good to know. I'll change that.